src/package/components/AddressView.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { useWeb3 } from "../"
+
+type Props = {
+    isShort?: boolean
+}
+
+/**
+ * AddressView is used to display the current address.
+ * It should be used inside Web3Provider so that it picks up address
+ * @param isShort - If true, the address is displayed in short format: 0xff...abcd
+ * @returns {JSX.Element}
+ */
+export const AddressView = ({ isShort = true }: Props): JSX.Element => {
```

Comment:
How to style `<AddressView />`? Consider wrapping in @mui `<Box />` and passing styling props: `sx`, `style`, `className`